library/spark/spark-library/src/main/java/uk/gov/gchq/gaffer/spark/operation/dataframe/GetDataFrameOfElements.java

```diff
@@ -40,10 +41,15 @@
     private SQLContext sqlContext;
     private List<Converter> converters;
 
-    public GetDataFrameOfElements() { }
+    public static class TypeReferenceDataSetRow extends TypeReference<Dataset<Row>> {
```

Comment:
Would we want any of these to be private? We could return an anonymous class instance in the "createOutputTypeReference()" method if so?